src/components/Captures/CaptureGallery.js

```diff
@@ -0,0 +1,281 @@
+      component="div"
+      count={captureCount || 0}
+      rowsPerPage={rowsPerPage}
+      page={page}
+      onChangePage={handleChangePage}
+      onChangeRowsPerPage={handleChangeRowsPerPage}
+      labelRowsPerPage="Captures per page:"
+    />
+  );
+
+  return (
+    <>
+      <Grid item className={classes.body}>
+        <Grid item className={classes.bodyInner}>
+          <Grid container>
+            <Grid
+              item
+              // ref={{ refContainer }}
+              style={{
+                width: '100%',
+              }}
+            >
+              <Grid
+                container
+                justify="space-between"
+                alignItems="center"
+                className={classes.title}
+              >
+                <Typography variant="h5">
+                  {captureCount !== null &&
+                    `${countToLocaleString(captureCount)} capture${
+                      captureCount === 1 ? '' : 's'
+                    }`}
+                </Typography>
+
+                <Grid className={classes.cornerTable}>
+                  <Button
+                    variant="text"
+                    color="primary"
+                    onClick={() => setShowGallery(false)}
+                    startIcon={<TableChartIcon />}
+                  >
+                    Table View
+                  </Button>
+                  {imagePagination}
+                </Grid>
+              </Grid>
+            </Grid>
+            <Grid item>
+              <Grid
+                container
+                className={classes.wrapper}
+                spacing={1}
+                data-testid="captures-gallery"
+              >
+                {captureImageItems}
+              </Grid>
+            </Grid>
+            <Grid item container justify="flex-end" className={classes.title}>
+              {imagePagination}
+            </Grid>
+          </Grid>
+        </Grid>
+      </Grid>
+      <SidePanel
+        onSubmit={handleSubmit}
+        submitEnabled={captures.length > 0}
+        variant="persistent"
+      />
+      {isApproveAllProcessing && (
+        <AppBar
+          position="fixed"
+          style={{
+            zIndex: 10000,
+          }}
+        >
+          <LinearProgress
+            color="primary"
+            variant="determinate"
+            value={complete}
+          />
+        </AppBar>
+      )}
+      {isApproveAllProcessing && (
+        <Modal open={true}>
+          <div></div>
+        </Modal>
+      )}
+      {false && !isApproveAllProcessing && capturesUndo.length > 0 && (
+        <Snackbar
+          open
+          autoHideDuration={15000}
+          ContentProps={{
+            className: classes.snackbarContent,
+            'aria-describedby': 'snackbar-fab-message-id',
+          }}
+          message={
+            <span id="snackbar-fab-message-id">
+              You have {isBulkApproving === true ? ' approved ' : ' rejected '}
```

Comment:
You can simplify the check to `{isBulkApproving ? ' approved ' : ' rejected '}` instead of doing 
`=== true`

Reply:
This code is not currently being used